### PR TITLE
Get only the raw features from server xml

### PR DESCRIPTION
**Pull request by TayyabDev**

### CODE REVIEW

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java [hunk 1]
```diff
@@ -663,11 +663,47 @@ public static String getShortName(ProvisioningFeatureDefinition fd) {
...
+                    if(el.getNodeName().equals("include")){
+                        String location = el.getAttribute("location");
+                        if(!newLocations.contains(location) && !visitedServerXmls.contains(location)){
+                            newLocations.add(location);
+                        }
+                    }
+                }
+
+            }
+//            NodeList includeList = element.getElementsByTagName("include");
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java:692]: Please remove the code commented out.

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java [hunk 2]
```diff
@@ -663,11 +663,47 @@ public static String getShortName(ProvisioningFeatureDefinition fd) {
...
+//                Node includeNode = includeList.item(i);
+//                System.out.println("node: " + includeNode.getNodeName());
+//                System.out.println("attrs : " + includeNode.getAttributes());
+//                Element includeElement = (Element) includeNode;
+//                String location = includeElement.getAttribute("location");
+//                if(!newLocations.contains(location) && !visitedServerXmls.contains(location)){
+//                    newLocations.add(location);
+//                }
+//            }
+            System.out.println("locations: " +newLocations);
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java:705]: Please replace system out with proper logging

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java [hunk 3]
```diff
@@ -679,8 +715,15 @@ public static String getShortName(ProvisioningFeatureDefinition fd) {
                 }
             }
         } catch (Exception e) {
-            throw new InstallException(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_INVALID_SERVER_XML", xml, e.getMessage()), e, InstallException.IO_FAILURE);
+            // do nothing
+//            throw new InstallException(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_INVALID_SERVER_XML", xml, e.getMessage()), e, InstallException.IO_FAILURE);
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallUtils.java:719]: Remove the code commented out

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#run-libby-bot

**Comment by wasbrdge** [comment]
This needs to be reverted since it's causing a hard break and perhaps more than one. See https://ibm-cloud.slack.com/archives/C30NGUQ9E/p1590084604261800 for details.

**Pull request merged**